libs/platform/src/lib/components/form/radio/radio.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Remove this file as there are not tests and issue and it will be tested  as part of RB Group 

- Can you move Radio under RadioGroup? I know logically in case of CheckBox, it makes sense to have it as extra folder, but I dont see a usecase somebody will be using fdp-radio on its own (they could use fd-radio), but our added value is RB Group.